app/serializers/user.js

```diff
@@ -5,7 +5,11 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (options && options.includeId) {
-      json.data.attributes = pick(json.data.attributes, ['email', 'password']);
+      if (snapshot.adapterOptions && snapshot.adapterOptions.updateMode === 'profile') {
```

Comment:
@niranjan94 Okay so we will send the whole model except `email` & `password` & let the server take care of the rest?